components/ui/SyntaxHighlight.vue

```diff
@@ -18,10 +20,10 @@ import { Component, Prop } from 'vue-property-decorator'
 export default class extends Vue {
   @Prop({ type: String, default: 'python' }) lang
   @Prop(String) label
+  @Prop({ type: String, default: '' }) code
```

Comment:
Non-blocking: add the missing types for this and other properties. If you don't want to make it optional cause you allow the empty string, use:
```suggestion
  @Prop({ type: String, default: '' }) code!: string
```